src/main/java/com/example/demo/EmailService.java

```diff
@@ -3,84 +3,47 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.mail.SimpleMailMessage;
 import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.JavaMailSenderImpl;
-import org.springframework.mail.javamail.MimeMessagePreparator;
-import org.springframework.scheduling.TaskScheduler;
 import org.springframework.scheduling.annotation.EnableScheduling;
 import org.springframework.scheduling.annotation.Scheduled;
-import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.stereotype.Service;
-
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Date;
 
+
 @Service
 @EnableScheduling
 public class EmailService {
 
-    Email email;
     SimpleMailMessage simpleMailMessage;
     JavaMailSender javaMailSender;
     ArrayList<Email> allEmailsList;
     WorkWithJsonFile jsonFileWork;
-    ArrayList<Email> current;
 
     @Autowired
     public EmailService(WorkWithJsonFile jsonFile,JavaMailSender javaMailSender) {
-        email=new Email();
         this.jsonFileWork=jsonFile;
-        allEmailsList=jsonFileWork.loadFromJsonEmailsList();
         this.javaMailSender=javaMailSender;
-        current=new ArrayList<Email>();
         simpleMailMessage=new SimpleMailMessage();
+        allEmailsList=jsonFileWork.getEmails();
     }
 
-    @Scheduled(fixedRate = 20000)
-    public void lookingForCurrentEmeils() {
-
-        //magic part of code with smell
-        Date finallyDate=null;
-        DateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy");
-        Date date = new Date();
-        String s1=dateFormat.format(date);
-        DateFormat dateFormatHours = new SimpleDateFormat("HH:mm");
-        Date hours=new Date();
-        String hoursS=dateFormatHours.format(hours);
-
-        try {
-             finallyDate=new SimpleDateFormat("dd-MM-yyyy.HH:mm")
-                    .parse(new String(s1)+"."+new String(hoursS));
-        } catch (ParseException e) {
-            e.printStackTrace();
-        };
 
+    @Scheduled(fixedRate = 20000)
+    public void lookingForCurrentEmails() {
 
         for (Email counter:allEmailsList){
-            if (counter.getDelieveryDate().compareTo(finallyDate)==0 && !counter.getSended() && !allEmailsList.contains(counter)){
-                current.add(counter);
+            if (/*counter.getDeliveryDate().equals(new Date()) && counter.getSended().equals(false)*/counter.getRecepientName().equals("alexanderpeterburg21@gmail.com") && !counter.getSended() ){
```

Comment:
if (counter.getDeliveryDate().equals(new Date()) && counter.getSended().equals(false)) instead of the provided)